streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/context/impl/LocalFileContextBackend.java

```diff
@@ -0,0 +1,49 @@
+package io.ray.streaming.runtime.context.impl;
+
+import io.ray.streaming.runtime.config.global.ContextBackendConfig;
+import io.ray.streaming.runtime.context.ContextBackend;
+import java.io.File;
+import org.apache.commons.io.FileUtils;
+
+public class LocalFileContextBackend implements ContextBackend {
```

Comment:
What case can't be vocered by memory backend and can be covered by local file beckend? I think both memory backend and local backedn cover the same scope. Lcoal file backedn should be removed.

Reply:
It will be used in tests.